docs/docs/api/rest/workflows/README.md

```diff
@@ -0,0 +1,247 @@
+# REST API example: Create an Workflow
+
+Here, we'll walk through an example of how to create a [workflow](/workflow/) programmatically using the [create workflow endpoint](/api/rest/#create-a-workflow) from a [shared link](/workflows/sharing/), and pass your own connected accounts, step and trigger props as configuration.
```

Comment:
You refer to “shared link” here but Share Link below. Consider choosing one and applying it consistently when referencing it here